lib/google_analytics/api.rb

```diff
@@ -0,0 +1,28 @@
+require 'rest_client'
+
+module GoogleAnalytics
+  class Api
+    def self.event(category, action, client_id = '555')
```

Comment:
Let's chuck it into a constant (`FALLBACK_CLIENT_ID`?) with a quick comment and then get this out :)